modules/ptp-dual-ports-oc.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/about-ptp.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ptp-dual-ports-oc_{context}"]
+= Using dual-port NICs to improve redundancy for PTP ordinary clocks
+
+{product-title} supports single and dual-port networking interface cards (NIC) as ordinary clocks for PTP timing. To improve redundancy, you can configure a dual-port NIC with one port as active and the other as standby.
+
+In this configuration, the ports in a dual-port NIC operate as follows:
+
+* The active port functions as an ordinary clock in the `Following` port state.
+
+* The standby port remains in the `Listening` port state.
+
+* If the active port fails, the standby port transitions to active to ensure continued PTP timing synchronization.
+
+If both ports become faulty, the clock state moves to the `HOLDOVER` state, then the `FREERUN` state when the holdover timeout expires, before resyncing to a leader clock.
+
+{product-title} supports PTP ordinary clocks on nodes with x86 or AArch64 architectures using the following dual-port NICs:
+
+* NVIDIA ConnectX-7 series
+
+* NVIDIA BlueField-3 series, in NIC mode
++
+[NOTE]
+====
+You must configure the NVIDIA BlueField-3 series DPU in NIC mode before configuring the interface as an ordinary clock with improved redundancy. For further information about configuring NIC mode, see link:https://docs.nvidia.com/networking/display/bluefielddpubspv422/modes+of+operation#src-141856548_ModesofOperation-NICModeforBlueField-3[NIC Mode for BlueField-3] (NVIDIA documentation).
```

Comment:
I would add the link to the reset procedure here as well as it is not clearly mentioned in the documentation. Something like "a reset is required after changing the card mode" and add the link

Reply:
Updated the source accordingly, thanks!